docs/user-guide/lns-configuration.md

```diff
@@ -53,8 +53,10 @@ environment variables:
 
 | Environment variable name | Description | Mandatory |
 | --- | --- | --- |
-| CLOUD_DEPLOYMENT | Switch to indicate that the LNS is deployed as a standalone cloud-based instance | true |
-| ENABLE_GATEWAY | Gateway mode is only applicable to IoT Edge-based deployments. Must be set to `false` or unset. | false |
-| IOTHUBHOSTNAME | Host name of the IoT Hub to which the LNS should connect to. | true |
-| HOSTNAME | Identifier of the LNS. | true |
-| REDIS_CONNECTION_STRING | Connection string used to connect to the deployed Redis instance. | true |
+| CLOUD_DEPLOYMENT | Switch to indicate that the LNS is deployed as a standalone cloud-based instance | Yes (must be set to true) |
+| ENABLE_GATEWAY | Gateway mode is only applicable to IoT Edge-based deployments. Must be set to `false` or unset. | No |
+| IOTHUBHOSTNAME | Host name of the IoT Hub to which the LNS should connect to. | Yes |
+| HOSTNAME | Identifier of the LNS. | Yes |
+| REDIS_CONNECTION_STRING | Connection string used to connect to the deployed Redis instance. | Yes |
+| FACADE_SERVER_URL | Azure Facade function url, e.g. "https://deployedfunction.azurewebsites.net/api" | Yes |
```

Comment:
Do we really want to duplicate the variables? On L. 49 - 51 we mention that we have some "shared" variables, should we maybe consider extracting them to a "common" section?

Reply:
I know that it might be redundant and potentially error prone on our side when we are making changes
At the same time, as a user I would probably jump directly to the table and definitely forgot about some shared but mandatory variables.

Let's keep them split for now, we can eventually rewrite the documentation in future versions